src/Cpp/1-getting-started/1-HelloWindow/Application.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Application.hpp"
+#include "Input/Input.hpp"
+
+#include <GLFW/glfw3.h>
+
+Application::Application(const std::string_view title)
+    : _input(nullptr), _window(nullptr), _width(0), _height(0), _title(title)
+{
+}
+
+Application::~Application()
+{
+    Application::Cleanup();
+}
+
+void Application::Run()
+{
+    if (!Initialize())
+    {
+        return;
+    }
+
+    while (!glfwWindowShouldClose(_window))
+    {
+        _input->Update(
+            static_cast<float>(_width) / 2.0f,
+            static_cast<float>(_height) / 2.0f);
+        glfwPollEvents();
+        Update();
+        Render();
+        glfwSwapBuffers(_window);
+    }
+}
+
+// ReSharper disable once CppMemberFunctionMayBeConst
```

Comment:
Comment seems out of place?

Reply:
Resharper sees that glfwTerminate can be wrapped in a static method, therefore asks that method to be made `static` which I dont agree to, hence that analysis hint